tests/cpp_check_version.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+zil_baseurl="https://raw.githubusercontent.com/Zilliqa/Zilliqa/master/src"
```

Comment:
This is a hack. How do you ensure that best code is in the master? Why don't you use tagged releases instead? Merged to master doesn't mean ready to release.